src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2567 @@
+    - 5: `info`
+    - 6: `verbose`
+    - 7: `debug`
+
+    By setting log.level, you can enable logging of all events with
+    severities above or equal to the given level.
+]])
+
+I['log.modules'] = format_text([[
+    Configure the specified log levels (`log.level`) for different modules.
+
+    You can specify a logging level for the following module types:
+
+    - Modules (files) that use the default logger.
+    - Modules that use custom loggers created using the `log.new()` function.
+    - The tarantool module that enables you to configure the logging level
+      for Tarantool core messages. Specifically, it configures the logging level
+      for messages logged from non-Lua code, including C modules.
+]])
+
+I['log.modules.*'] = format_text([[
+    The log level.
+
+    For example: you have module placed by the following path:
+    `test/module.lua`. To configure logging levels, you need to
+    provide module names corresponding to paths to these modules:
+    `test.module: 'verbose'`.
+]])
+
+I['log.nonblock'] = format_text([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+I['log.pipe'] = format_text([[
+    Start a program and write logs to its standard input (`stdin`). To send
+    logs to a program's standard input, you need to set `log.to` to `pipe`.
+]])
+
+I['log.syslog'] = format_text([[
+    Syslog configurations parameters. To write logs to syslog,
+    you need to set `log.to` to `syslog`.
+]])
+
+I['log.syslog.facility'] = format_text([[
+    Specify the syslog facility to be used when syslog is enabled.
+    To write logs to syslog, you need to set `log.to` to `syslog`.
+]])
+
+I['log.syslog.identity'] = format_text([[
+    Specify an application name used to identify Tarantool messages in
+    syslog logs. To write logs to syslog, you need to set `log.to` to `syslog`.
+]])
+
+I['log.syslog.server'] = format_text([[
+    Set a location of a syslog server. This option accepts one of the following
+    values:
+
+    - An address. Example: `127.0.0.1:514`.
+    - A Unix socket path starting with `unix:`.
+      Examples: `unix:/dev/log` on Linux or `unix:/var/run/syslog` on macOS.
+
+    To write logs to syslog, you need to set `log.to` to `syslog`.
+]])
+
+I['log.to'] = format_text([[
+    Define a location Tarantool sends logs to. This option accepts the
+    following values:
+
+    - `stderr`: write logs to the standard error stream.
+    - `file`: write logs to a file.
+    - `pipe`: start a program and write logs to its standard input.
+    - `syslog`: write logs to a system logger.
+]])
+
+-- }}} log configuration
+
+-- {{{ lua configuration
+
+I['lua'] = format_text([[
+    This section defines configuration parameters related
+    to the Lua within Tarantool.
```

Comment:
Nit, I haven't spotted it before:
```suggestion
    to Lua within Tarantool.
```

Reply:
Fixed.